pkg/api/api_test.go

```diff
@@ -0,0 +1,20 @@
+package api_test
```

Comment:
nope. This is the test suffix only... This makes sure that you don't load test functions to your binary when compiling